index.html

```diff
@@ -45,14 +58,16 @@
 </header>
 
 <main>
-  <article id = "post">
+  <div id="app">
+  </div>
+  <!-- <article id = "post">
     <h1 class = "postTitle"></h1>
     <p class = "postAuthor"></p>
     <p class = "postCategory"></p>
     <div class = "postBody"></div>
     <p class = "postAge"></p>
     <a href = "#" class = "read-on">Expand/Collapse</a>
```

Comment:
Try and avoid committing commented out code to the master branch unless there is a reason to have it. Best would be to explain the reason if so. Maybe only have `script id="blogArticle" inserted here` or something like that.